<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1147</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="2,0,3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,10,4">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>文件名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="filename_cbox"/>
        </item>
        <item>
         <widget class="QPushButton" name="preview_button">
          <property name="text">
           <string>预览</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,10,4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>截取时间段</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sub_line">
          <property name="placeholderText">
           <string>请输入起始时间和结束时间 如（0 10.5）</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sub_button">
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="silence_button">
          <property name="text">
           <string>静音</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="back_button">
          <property name="text">
           <string>倒放</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pic_music_button">
          <property name="text">
           <string>提取音频</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="x_mirror_button">
          <property name="text">
           <string>x轴镜像</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="y_mirror_button">
          <property name="text">
           <string>y轴镜像</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QComboBox" name="speed_cbox">
          <item>
           <property name="text">
            <string>0.1倍速</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0.5倍速</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1.5倍速</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2倍速</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4倍速</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8倍速</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="speed_button">
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,10,4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>画面大小</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="size_line">
          <property name="placeholderText">
           <string>请输入画面长宽 如（1920 1080）</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="size_button">
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="3,10,4">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>音量</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="volumex_line">
          <property name="placeholderText">
           <string>请输入音量大小 如（0.5）</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="volumex_button">
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="3,10,4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>画面明暗</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="light_line">
          <property name="placeholderText">
           <string>请输入明暗比 如（1.5）</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="light_button">
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="3,10,4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>视频连接</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="connect_line">
          <property name="placeholderText">
           <string>请输入视频序号 如（1 2 3）</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="connect_button">
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="3,10,4">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>音频拼接</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="music_connect_line">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>请输入音频序号 如（1 2 3）</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="music_connect_button">
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="3,10,4">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>添加背景音</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inc_music_line">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>请输入音频序号（单个或多个）</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="inc_music_button">
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_10">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="3,10,4">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>图片转视频</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pic2video_line">
          <property name="placeholderText">
           <string>请输入视频帧数，默认25</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pic2video_button">
          <property name="text">
           <string>选择并转换</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_11">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="3,10,4">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>视频转图片</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="video2pic_line">
          <property name="placeholderText">
           <string>请输入帧数，默认每帧导出</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="video2pic_button">
          <property name="text">
           <string>选择并转换</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_13">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="3,4,6,4">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>视频拼接</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="clips_cbox">
          <item>
           <property name="text">
            <string>水平拼接</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>垂直拼接</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>整体拼接</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="clips_line">
          <property name="placeholderText">
           <string>请输入视频序号 如（1 2 3）</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clips_button">
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_12">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>字幕</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="1,1,1,1">
           <item>
            <widget class="QComboBox" name="font_size_cbox">
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>10</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>12</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>14</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>16</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>18</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>20</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>22</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>24</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>26</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>28</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>30</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>32</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>34</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>36</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>38</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>40</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>42</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>44</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>46</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>48</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>50</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>52</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>54</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>56</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>58</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>60</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>62</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>64</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>66</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="pos_cbox">
             <item>
              <property name="text">
               <string>top</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>center</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>bottom</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>right</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>left</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="fontcolor_button">
             <property name="text">
              <string>颜色</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="font_line">
             <property name="placeholderText">
              <string>请输入字幕开始时间和结束时间</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14"/>
         </item>
         <item>
          <widget class="QTextEdit" name="font_text">
           <property name="placeholderText">
            <string>请输入字幕……</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="font_insert_button">
           <property name="text">
            <string>插入</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_14">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>文件信息</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="file_info"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1147</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionopen"/>
    <addaction name="actionsave"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionopen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
